nvtabular/ops/add_metadata.py

```diff
@@ -39,3 +40,38 @@ def output_tags(self):
     @property
     def output_properties(self):
         return self.properties
+
+
+class AddTags(AddMetadata):
+    def __init__(self, tags=None):
+        super().__init__(tags=tags)
+
+
+class AddProperties(AddMetadata):
+    def __init__(self, properties=None):
+        super().__init__(properties=properties)
+
+
+# Wrappers for common features
+class TagAsUserID(Operator):
+    @property
+    def output_tags(self):
+        return [Tags.USER_ID]
+
+
+class TagAsItemID(AddTags):
```

Comment:
I think these next few classes should just subclass `Operator` like the first one

Reply:
apologies, that was an oversight on my part, fixed this now

Reply:
No worries! You're taking tickets off my queue, so I'm not going to complain. 😀